File of SQL Commands

abhisheks008 opened the pull request:
In this file I have attached all the SQL commands those are needed to Relational Database Management Systems.

Project Name : File of SQL Commands
Fix #86 

@Niketkumardheeryan Please check it!

Thank you!

abhisheks008 commented:
@Niketkumardheeryan I have added the file of SQL Commands, where I have talked about all those commands and also I have provided the syntax and example for each command to make it easier to understand. I have used the markdown file extension here so that all those highlights and the headings can be done which will make the file better to understand. Also I have added the .txt file for those who have issues with the markdown file.

Please check it and let me know.

Niketkumardheeryan commented:
@abhisheks008 add .dox or pdf file or add more examples, and  add joining  of tables too.

abhisheks008 commented:
@Niketkumardheeryan Updated all the required changes.

- [x] Provided more examples.
- [x] Added the JOINs, CLAUSEs in the file.
- [x] Converted the file into PDF and re-uploaded that.

Please check it now.

Thank you!
